dev.Dockerfile

```diff
@@ -28,19 +25,13 @@ ENV UV_LINK_MODE=copy
 RUN --mount=type=cache,target=/root/.cache/uv \
     --mount=type=bind,source=uv.lock,target=uv.lock \
     --mount=type=bind,source=pyproject.toml,target=pyproject.toml \
-    uv sync --frozen --no-install-project --no-dev
+    uv sync --frozen --no-install-project && uv pip install pip
```

Comment:
uhmm the reason is to avoid running pip install by mistake thinking it will install within the venv create by uv. Probably using an alias is a better approach.